charmcraft.yaml

```diff
@@ -1,20 +1,8 @@
 type: charm
-bases:
-  - build-on:
-      - name: ubuntu
-        channel: "20.04"
-    run-on:
-      - name: ubuntu
-        channel: "20.04"
-        architectures: [amd64]
-  - build-on:
-      - name: ubuntu
-        channel: "22.04"
-    run-on:
-      - name: ubuntu
-        channel: "22.04"
-        architectures: [amd64]
 
+platforms:
+  ubuntu@20.04:amd64:
+  ubuntu@22.04:amd64:
```

Comment:
need to use short hand notation. See https://github.com/canonical/data-platform-workflows/issues/268